Sources/Persistence/DiskMemoryPersistenceStack.swift

```diff
@@ -193,7 +205,7 @@ public final class DiskMemoryPersistenceStack: NSObject, PersistenceStack {
             }
 
             completion { [weak self] in
-                self?.performanceMetrics?.end(with: identifier)
+                self?.persistencePerformanceMetrics?.metrics.end(with: identifier)
```

Comment:
shouldn't/couldn't we also associate the blob's size?

Reply:
Fixed